functions/plot_function.r

```diff
@@ -0,0 +1,12 @@
+# Function creating a pretty plot
+plotTraffic <- function(data, name) {
```

Comment:
Very nice that you made the function in a separate file like this. You did however forget to add the following line to the `iterations.r` file:
```
source("functions/plot_function.r")
```